Makefile.am

```diff
@@ -116,50 +111,18 @@ osx_volname:
 	echo $(OSX_VOLNAME) >$@
 
 if BUILD_DARWIN
-$(OSX_DMG): $(OSX_APP_BUILT) $(OSX_PACKAGING) $(OSX_BACKGROUND_IMAGE)
```

Comment:
One possible simplification after this change, would be to make the `otool` and `installnametool` checks no longer cross-compiling dependent. Then we should be able to have a single `OSX_ZIP` rule, and drop the need for `BUILD_DARWIN`.